src/tests/Microsoft.PowerFx.Interpreter.Tests/Microsoft.PowerFx.Interpreter.Tests.csproj

```diff
@@ -24,6 +24,7 @@
     </ItemGroup>
 
     <ItemGroup>
+      <ProjectReference Include="..\..\libraries\Microsoft.PowerFx.Connectors\Microsoft.PowerFx.Connectors.csproj" />
```

Comment:
>  <ProjectReference Include="..\..\libraries\Microsoft.PowerFx.Connectors\Microsoft.PowerFx.Connectors.csproj" /> [](http://example.com/codeflow?start=4&length=113)

Let's avoid this . It breaks the layering. 
(imagine that we eventually move connector csproj + test into its own repo)

Reply:
Correct !